openassessment/workflow/workflow_batch_update_api.py

```diff
@@ -228,23 +228,26 @@ def update_workflow_for_submission(submission_uuid, assessment_requirements=None
         raise UpdateWorkflowForSubmissionException(str(e)) from e
 
 
-def is_flexible_peer_grading_on(ora_block, course_block):
+def is_flexible_peer_grading_on(assessment_requirements, course_settings):
```

Comment:
This and all tests for it can be removed, and you can use `openassessment.assessment.api.peer.flexible_peer_grading_enabled` It already has this logic, is tested, and has the identical signature

Reply:
The signature is the same but slightly different logic. if ORA peer grading or flexible peer grading is not defined existing api will return `None`.